headers/types/dungeon_mode/dungeon.h

```diff
@@ -1176,7 +1180,7 @@ struct dungeon {
     undefined field_0x3e19;
     undefined field_0x3e1a;
     undefined field_0x3e1b;
-    undefined4 field_0x3e1c;
+    undefined field_0x3e1c;
```

Comment:
Why was this changed? It's going to break the size of the whole struct.

Reply:
I thought I made a mistake typing because it's directly above one of my comments. Will correct.